docs/dml-of-carbondata.md

```diff
@@ -316,12 +311,12 @@ CarbonData DML statements are documented here,which includes:
   INSERT OVERWRITE TABLE table1 SELECT * FROM TABLE2
   ```
 
-### INSERT DATA INTO CARBONDATA TABLE From Stage Input Files
+## INSERT DATA INTO CARBONDATA TABLE From Stage Input Files
 
   Stage input files are data files written by external application (such as Flink). These files 
   are committed but not loaded into the table. 
   
-  You can use this command to insert them into the table, so that making them visible for query.
+  User can use this command to insert them into the table, so that making them visible for a query.
```

Comment:
"User can use this command to insert them into the table, so that making them visible for a query." can be changed to "User can use this command to insert them into the table, thus making them visible for a query."

Reply:
done